scans.go

```diff
@@ -472,6 +472,22 @@ func (c Cx1Client) ScanProjectGitByID(projectID, repoUrl, branch string, setting
 	return scan, err
 }
 
+func (c Cx1Client) ScanProjectSHAByID(projectID string, handler ScanHandler, settings []ScanConfiguration, tags map[string]string) (Scan, error) {
```

Comment:
```suggestion
// convenience function
func (c Cx1Client) ScanProjectGitByIDWithHandler(projectID string, handler ScanHandler, settings []ScanConfiguration, tags map[string]string) (Scan, error) {
```

The function is a variation of ScanProjectGitByID. The change is accepting a handler as a parameter. The function is not limited to scan by sha. It can work exactly the same as ScanProjectGitByID and scan by referencing a branch name.